scripts/rff/historical_weather_variability/county/R/county_aggregation.R

```diff
@@ -0,0 +1,248 @@
+### Aggregate PRISM to county ###
+rm(list=setdiff(ls(), c("master_path", "code_path")))
+path <- master_path
+
+coords <- read.csv(paste0(path, "replication/data/monthly_dataframes/month_ppt_coordinates.csv"))
+
+counties <- tigris::counties(cb = T)
+counties <- as(counties, 'Spatial')
+
+sp <- SpatialPoints(coords)
+proj4string(sp) <- proj4string(counties)
+  
+counties <- crop(counties, extent(sp))
+over <- over(sp, counties)
+
+coords$fips <- over$GEOID
+
+write.csv(coords, paste0(path, "replication/data/prism_coordinates_fips.csv"), row.names = F)
+
+
+#### RANGES ####
+
+# TMIN
+load(paste0(path, "replication/data/ranges/tmin/tmin_ranges_dataframe.Rdata"))

```

Comment:
For this and the sections below, it might be better to abstract the procedure in separate helper functions. Eventually, after every call to a helper function returns, the merge() function can be used..

Reply:
Marking resolved to submit.